Makefile

```diff
@@ -0,0 +1,27 @@
+PUML_URLS_PATTERN=http://www\.plantuml\.com.*develop/docs/\(.*\).puml&fmt=svg&vvv=1&sanitize=true
+PUML_URLS_REPLACE=https://adorsys.github.io/open-banking-gateway/doc/${TRAVIS_TAG}/\1.png
+
+site: clean_docs replace_puml_urls convert_puml build_docs copy_puml
+
+.PHONY : replace_puml_urls
+replace_puml_urls:
+	cp README.md docs/README.md
+	sed -i 's/docs\///g' docs/README.md
+	find . -type f -name "*.md" -print -exec sed -i.bak 's%${PUML_URLS_PATTERN}%${PUML_URLS_REPLACE}%' {} \;
+	find . -type f -name "*.md.bak" -print -delete
```

Comment:
Not sure, I think -i is an apple OSX specific option. And if you want to delete the bakups anyway you can do:
sed -i "" s/....